contracts/FountainV1.sol

```diff
@@ -307,7 +266,22 @@ contract FountainV1 {
         _updateTrackedRedistribution(currentMoneyPool);
 
         // Emit events.
-        emit SustainMoneyPool(moneyPoolId, msg.sender, amount);
+        emit SustainMoneyPool(
+            moneyPoolId,
+            currentMoneyPool.owner,
+            msg.sender,
+            amount
+        );
+
+        if (wasInactive)
+            // Emit an event since since is the first sustainment being made towards this MoneyPool.
+            emit ActivateMoneyPool(
```

Comment:
It will be interesting to analyze gas costs. For example, the first transaction to sustain an address will pay for two events, while others will only pay for one.

Reply:
Thats a great thought. I added a note to the block